File: MyApplication19/app/src/main/java/com/example/myapplication19/LanguageViewHolder.java
package com.example.myapplication19;

import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

public class LanguageViewHolder extends RecyclerView.ViewHolder {
    TextView orderNumber;
    TextView language;
    Button btnDelete;

    public LanguageViewHolder(@NonNull View itemView) {
        super(itemView);
        orderNumber = itemView.findViewById(R.id.tvOrderNumber);
        language = itemView.findViewById(R.id.tvLanguage);
        btnDelete = itemView.findViewById(R.id.btnDelete);
    }
}

File: MyApplication18/app/src/main/java/com/example/myapplication/adapter/LanguageViewHolder.kt
package com.example.myapplication.adapter


import android.view.View
import android.widget.Button
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.example.myapplication.R

/**
 * Created by Maulana Ibrahim on 15/August/2020
 * Email maulibrahim19@gmail.com
 */
class LanguageViewHolder(v: View) : RecyclerView.ViewHolder(v) {
    val orderNumber: TextView = v.findViewById(R.id.tvOrderNumber)
    val language: TextView = v.findViewById(R.id.tvLanguage)
    val btnDelete:Button = v.findViewById(R.id.btnDelete)
}